wiki/Game_modifier/Double_Time/ko.md

```diff
@@ -0,0 +1,62 @@
+---
+tags:
+  - double time
+  - double-time
+  - DT
+  - mod
+  - game modifier
+  - 더블 타임
+---
+
+# 더블 타임 (모드)
+
+![DT mod icon](/wiki/shared/mods/DT.png "더블 타임 모드 아이콘")
+
+*모든 모드의 목록을 확인하려면 [여기](/wiki/Game_modifier)를 클릭해 주세요.*\
+*관련 모드: [Nightcore](/wiki/Game_modifier/Nightcore)*
+
+## 더블 타임 모드
+
+- 축약형: DT
+- 종류: 난이도 상승 모드
+- 점수 배수:
+  - ![][o!s]: 1.12x
+  - ![][o!t]: 1.12x
+  - ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- 기본 단축키: `D`
+- 설명: `빨라아아아아아아아!!`
+- DT를 사용할 수 있는 osu!의 장르 ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- 같은 계열의 모드: [Nightcore](/wiki/Game_modifier/Nightcore)
+
+## 상세 설명
+
+*중요: 이 모드는 노래의 속도를 증가시키지만, 노래의 높낮이를 바꾸지 않습니다.*
+
+**더블 타임** 모드는 맵의 속도를 1.5배 증가시킵니다. 또한 노래의 길이를 33% 줄이고 [전체적인 난이도(OD)](/wiki/Beatmapping/Overall_difficulty)와 [체력 감소량(HP)](/wiki/Beatmapping/Health)을 높이며 [원이 생기고 사라지기까지 걸리는 시간(AR)](/wiki/Beatmapping/Approach_rate)을 감소시키는 [모드](/wiki/Game_modifier)입니다.
+
+더블 타임 모드는 ![][o!s]에서 쉬운 난이도의 맵을 플레이할 때 더 많은 [pp](/wiki/Performance_points)를 얻을 수 있는 가장 좋은 모드로 널리 이용되고 있습니다.
```

Comment:
should replace the osu!standard icon with an actual link to the article

line for reference:

```md
The Double Time mod is widely regarded as one of the best mods used for gaining large amounts of [performance points](/wiki/Performance_points) on easier difficulties in [osu!standard](/wiki/Game_mode/osu!).
```

Reply:
All done :)